docs/application/dotnet/guides/tizen.circularUI/CircleStepper.md

```diff
@@ -0,0 +1,108 @@
+You can modify the value with [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+To receive [Rotary event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary), it must be registered as `RotaryFocusObject`, property of [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage).
+
+![](data/CircleStepper_property.png)
+
+**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be confined in the `CirclePage` container or [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior) must be added in [Behaviors](https://developer.xamarin.com/api/type/Xamarin.Forms.Behavior/) class of [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) class that contains these controls. If other `pages` contain these controls, it may cause exception or cannot display the controls.**
+
+## To add CircleStepper in the CirclePage
+
+You can set `CircleStepper` in the [CirclePage.Content](xref:Tizen.Wearable.CircularUI.doc.CirclePage). For more information on how to add a [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage), see [CirclePage guide](https://samsung.github.io/Tizen.CircularUI/guide/CirclePage.html#create-circlepage).
+The following code shows [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage) with `CircleStepper`.
+`RotaryFocusTargetName` property sets the currently focused control using the rotating interaction, and displays the focused control's circle object.
+If you don't set this value properly, control can't receive [Rotary Event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary).
+
+`CircleStepper` has the following properties:
+
+- LabelFormat : This property gets or sets the format in which the value is shown.
+- MarkerColor : This property changes the color of the marker to the value selected.
+- MarkerLineWidth : This property gets and sets the length of the marker.
+
+This example consists of `StackLayout` with a `CircleStepper` and two labels in the `CirclePage`.
+Since 9 is set to `Minimum` and `LabelFormat` is set to "% 1.1f", "9.0" appears on the screen. Since the `Increment` is 7.5, it increases by 7.5 when you turn the bezel and increase the `Value`. The `Maximum` defined value is 99, hence the marker does not exceed the value. Since the `MarkerColor` is set to "Coral", it will be displayed as the corresponding color.
+
+For more information, see below links.
```

Comment:
For more information, see below links.
Please change to " For more information, see the following links: